### PR TITLE
Qiangliu/dzil for locale

### PR DESCRIPTION
**Pull request by nooneisthere**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,22 @@
+sudo: false
```
Comment (zakame) [inline .travis.yml]: `sudo` is no longer needed here per https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures (i.e. it is already a no-op for the most part)

File: .travis.yml [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
-    - "5.18"
-    - "5.16"
-    - "5.14"
-    - "5.12"
+   - '5.24'
+   - '5.22'
+   - '5.20'
+   - '5.18'
+   - '5.16'
+   - '5.14'
```
Comment (zakame) [inline .travis.yml]: I'd recommend updating to

```yaml
perl:
   - "5.28"
   - "5.26"
   - "5.24"
   - "5.14"
```

As the last one is the earliest we can care to support, while the latest 3 are currently supported per https://metacpan.org/pod/perlpolicy#MAINTENANCE-AND-SUPPORT

File: cpanfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+requires 'Locale::Country';
+requires 'Locale::Country::Multilingual';
+requires 'perl', '5.006';
```
Comment (zakame) [inline cpanfile]: I think `perl >= 5.6` is implied, could be dropped here

File: cpanfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+on configure => sub {
+    requires 'ExtUtils::MakeMaker';
```
Comment (zakame) [inline cpanfile]: Would be useful here if we're depending on a specific minimum MakeMaker version; otherwise this can be dropped

File: cpanfile [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+};
+
+on build => sub {
```
Comment (zakame) [inline cpanfile]:
```suggestion
on test => sub {
```

File: .travis.yml (hunk 2 above)
Comment (nooneisthere) [reply .travis.yml]: OK, should we also update this doc ?https://github.com/regentmarkets/cpan/wiki/Using-dzi
l

File: cpanfile (hunk 4 above)
Comment (nooneisthere) [reply cpanfile]: this cpanfile is generated auto by ` mymeta-cpanfile > cpanfile`
should we change it manually?

File: .travis.yml (hunk 2 above)
Comment (nooneisthere) [reply .travis.yml]: I updated it ,seems traivs fail on 5.28

File: .travis.yml (hunk 1 above)
Comment (nooneisthere) [reply .travis.yml]: removed

File: cpanfile (hunk 4 above)
Comment (zakame) [reply cpanfile]: Yes, `mymeta-cpanfile` is a transition tool mostly, maintainers should be updating `cpanfile` directly.

File: cpanfile (hunk 5 above)
Comment (chylli) [reply cpanfile]:
```t/00-compile.t ......... 1/2 Locale::Country will be removed from the Perl core distribution in the next major release. Please install it from CPAN. It is being used at /home/travis/build/binary-com/perl-Locale-Country-Extra/.build/e8kC7i0vbb/blib/lib/Locale/Country/Extra.pm, line 7.
Locale::Codes will be removed from the Perl core distribution in the next major release. Please install it from CPAN. It is being used at /home/travis/perl5/perlbrew/perls/5.28.1/lib/5.28.1/Locale/Country.pm, line 22.```
So I guess we need to add Local::Country into cpanfile and install it by cpm if necessary.

File: .travis.yml (hunk 2 above)
Comment (nooneisthere) [reply .travis.yml]: t/00-compile.t ......... 1/2 Locale::Country will be removed from the Perl core distribution in the next major release
Locale::Codes will be removed from the Perl core distribution in the next major release. 

we got those waring for use perl 5.28, should we remove 5.28?

File: cpanfile (hunk 4 above)
Comment (nooneisthere) [reply cpanfile]: updated

**Pull request merged**